userappwithbroker/build.gradle

```diff
@@ -0,0 +1,23 @@
+apply plugin: 'com.android.application'
+
+android {
+    compileSdkVersion 21
+    buildToolsVersion "23.0.3"
```

Comment:
:) shouldn't be 23, we should only update it when sdk version gets bumped to 23.